src/engine/fox_hud.c

```diff
@@ -37,10 +37,10 @@ s32 D_80161794;
 s32 D_80161798;
 f32 D_8016179C;
 f32 D_801617A0;
-f32 D_801617A4;
-f32 D_801617A8;
-f32 D_801617AC;
-s32 D_801617B0;
+f32 shieldGaugeDesiredScale;
+f32 shieldGaugeCurrentScale;
+f32 shieldFillAmount;
+s32 shieldUpgradeTimer;
```

Comment:
These are static vars
```suggestion
sShield....